package-lock.json

```diff
@@ -1,15806 +1,8 @@
 {
   "name": "bonfire-frontend",
   "version": "0.1.0",
-  "lockfileVersion": 2,
```

Comment:
I just noticed this after I finished my review. You seem to be on an older version of NodeJS (and therefore NPM). You can check with `npm --version` and `node --version`. Using an older version of the lockfile isn't really ideal so please update your installation. I recommend installing the latest _LTS_ version of NodeJS from [here](https://nodejs.org/en/).

Reply:
I'm not sure why it says that this change request comment is outdated. Ignore that little marker until the file has actually been changed.